Textile/Classes/LifecycleManager.m

```diff
@@ -118,7 +118,10 @@ - (void)stopNodeAfterDelay:(NSTimeInterval)delay {
   }
   self.timer = [NSTimer scheduledTimerWithTimeInterval:delay repeats:FALSE block:^(NSTimer * _Nonnull timer) {
     [weakSelf stopNode];
-    [UIApplication.sharedApplication endBackgroundTask:bgTaskId];
+    // Dispatch this after a slight delay to allow consumers of the stop node event to process the event before app suspension
```

Comment:
Been meaning to get this in here. I think it will allow clients, especially via the react-native-sdk, to not have the app suspended before having time to process the node stopped event. This should eliminate those cases where you sometimes don't see the verbose UI node stop notification until you bring the app back into the foreground.